### PR TITLE
UAG-303 Fix: Display Conditions not working on Woocommerce Pages on Frontend.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- Card - https://ua-gutenberg.atlassian.net/jira/software/projects/UAG/boards/2?selectedIssue=UAG-303

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Added in a card.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: classes/class-uagb-post-assets.php [hunk 1]
```diff
@@ -247,10 +247,20 @@ public function allow_assets_generation() {
...
 	public function enqueue_scripts() {
 
-		// Global Required assets.
-		if ( has_blocks( $this->post_id ) ) {
-			/* Print conditional css for all blocks */
-			add_action( 'wp_head', array( $this, 'print_conditional_css' ), 80 );
+		$this_post = UAGB_Helper::get_woocommerce_post_object();
+
+		if ( class_exists( 'WooCommerce' ) && ! empty( $this_post ) && is_object( $this_post ) ) {
+			// Assets Required for woocommerce pages.
```
Comment (RajkiranBagal) [inline classes/class-uagb-post-assets.php:253]: We can combine these 2 if statements.

File: classes/class-uagb-post-assets.php [hunk 2]
```diff
@@ -247,10 +247,20 @@ public function allow_assets_generation() {
...
+			if ( has_blocks( $this_post ) ) {
+				/* Print conditional css for all blocks */
+				add_action( 'wp_head', array( $this, 'print_conditional_css' ), 80 );
+			}
+		} else {
+			// Global Required assets.
```
Comment (RajkiranBagal) [inline classes/class-uagb-post-assets.php:259]: We can combine this else & if statement

File: classes/class-uagb-helper.php [hunk 3]
```diff
@@ -1604,6 +1580,45 @@ public function generate_assets() {
 				}
 			}
 		}
+
+		/**
+		 * Check is it woocommerce object or not.
+		 *
+		 * @since x.x.x
+		 * @deprecated x.x.x
```
Comment (RajkiranBagal) [inline classes/class-uagb-helper.php]: Remove deprecated tag

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: classes/class-uagb-post-assets.php (hunk 1 above)
Comment (AbhijitNage123) [reply classes/class-uagb-post-assets.php]: Updated please check

File: classes/class-uagb-post-assets.php (hunk 2 above)
Comment (AbhijitNage123) [reply classes/class-uagb-post-assets.php]: Updated please check

File: classes/class-uagb-helper.php (hunk 3 above)
Comment (AbhijitNage123) [reply classes/class-uagb-helper.php]: Done

**Review by RajkiranBagal — APPROVED** [review]

**Comment by RajkiranBagal** [comment]
Closing this PR as this issue get's fixed here - https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/811

**Pull request closed without merging**